nsfc-plugin/bnd.bnd

```diff
@@ -1,8 +1,7 @@
-Meta-Persistence: META-INF/persistence.xml
 
-#These are needed due to a bug in Hibernate
+Bundle-SymbolicName: sdn-controller-nsfc-plugin
 
```

Comment:
question: symbolic name should have been there earlier as well? even without SFC support?

Reply:
Ever bundle has a symbolic name, but before it was identical to the maven module name. Now we have two modules. One of them is called nsfc-plugin (the other is the uber-bundle)  but teh symbolic name needs to stay the same as before.